collaboration/1.slack.md

```diff
@@ -0,0 +1,101 @@
+# Slack
+
+[Slack](https://slack.com/) is a chat we use to discuss about anything.
```

Comment:
Maybe [this](https://slack.com/is) would be the best link then, but that URL could change without notice, so I'm fine with https://slack.com